go/example.go

```diff
@@ -96,47 +90,55 @@ func main() {
 	client, err := flight.NewClientWithMiddleware(
 		net.JoinHostPort(config.Host, config.Port),
 		nil,
-		[]flight.ClientMiddleware{
-			flight.NewClientCookieMiddleware(),
-		},
+		[]flight.ClientMiddleware{flight.NewClientCookieMiddleware()},
 		grpc.WithTransportCredentials(creds),
 	)
 	if err != nil {
 		log.Fatal(err)
 	}
 	defer client.Close()
 
+	// TestableFlightClient is simple wrapper for flight.Client to allow for testing
+	testableClient := &implementations.TestableFlightClient{Client: client}
```

Comment:
do we still need then &implementations.TestableFlightClient ?
Can we just pass normal client inside run now?